R/calcDiffInvestCosts.R

```diff
@@ -124,28 +124,33 @@ calcDiffInvestCosts <- function(subtype) {
     # rooftop is already well-established and has prices around 1.25$/W.
     # Accordingly, we expect that by 2020, the utility-scale solar market will be more
     # in equilibrium and have prices below the current rooftop prices
-    x_adj["CAZ", , ] <- 1200
+    # Also, the 2023 Australia PVPS report states ~1.5AUSD/W for utility-scale for 2018-2022,
+    # equivalent to 1.05USD2020/W, and the Canadian PVPS report has 1.3CanD/W = 1USD/W
+    x_adj["CAZ", , ] <- 1000
+    # The Chinese PVPS report states ~3.5RMB/W, ~0.5$/W for utility-scale, in 2022
+    x_adj["CHA", , ] <- 600
     # IND: Other sources for Indian utility-scale prices (IRENA, WEO, REN21) are more in the range of ~800$/kW.
     # Also, the reports around failed auctions and non-delivery of projects might indicate that
     # the stated prices are below cost.
     x_adj["IND", , ] <- 700
-    # JPN: We assume that Japan prices will be downward-influenced by the low prices realized everywhere else in Asia.
-    x_adj["JPN", , ] <- 1500
+    # JPN: We assume that Japan prices will be downward-influenced by the low prices realized everywhere else in Asia. Also, the 2022 PVPS
+    # report for JPN shows ~120-130 JPY/W, roughly 1.1US$2020/W
+    x_adj["JPN", , ] <- 1100
     # LAM: IRENA states utility-scale prices in the range of ~1400-1500$/kW for LAM.
     # Recent auctions in individual countries indicate lower prices, but it is unlikely that
     # all the countries will immediately achieve the lowest prices realized in some auctions
-    x_adj["LAM", , ] <- 950
+    x_adj["LAM", , ] <- 850
     # MEA: IRENA states utility-scale prices in the range of ~1250$/kW for MEA
     # Recent auctions in individual countries indicate lower prices,
     # but it is unlikely that all the countries will immediately achieve
     # the lowest prices realized in some auctions
-    x_adj["MEA", , ] <- 850
+    x_adj["MEA", , ] <- 800
     # REF: Very different costs in IRENA (2300) and REN21 (1300) -
     # but unlikely to have higher capital costs than severly space-constrained Japan
     x_adj["REF", , ] <- 1400
```

Comment:
The comment says that for capital cost REF < JPN, but the new values don't match that (1400 vs 1100).
Which one is correct:
- REF may have higher costs than JPN?
- REF costs are below 1100?

Reply:
@robertpietzcker

Reply:
Good catch, Fabrice : the comment on REF was written with respect to the old values from 2018, and the change I now made to JPN was based on newer IEA PVPS data for JPN. 

I know little about REF PV costs. 

I see two options: 
a) leaving REF as is, and adding a "much" in the sentence, as in "unlikely to have much higher capital costs"
b) reducing REF to 1100. 

any thoughts on this, @lecfab ?

Reply:
I'm also very ignorant on REF PV costs.
There is this [arxived paper](https://arxiv.org/pdf/2312.07240) (trying to assess whether russian pv can go without subsidies) that shows the following costs. Not sure what they exactly encompass. As of today, 1RUB ≃ 0.01$. The capital cost would range between 600 and 1200 $/kW.

![image](https://github.com/user-attachments/assets/2a1a4841-ecff-406e-a280-6a56cf242d6f)

A few examples in Kazakhstan seem to get capital costs between [500 ](https://mergerscorp.com/property/20mw-kazakh-pv-photovoltaic-power-plant-gov-ppa/) and [850](https://renewablemarketwatch.com/country-reports/renewable-projects-database/solar-pv-projects/kazakhstan-35-mwp-solar-photovoltaic-pv-power-plant-with-building-permit-project-profile-single-user-detail) $/kW. I am not sure these sources are reliable, and I don't know if everyone means the same thing with capital cost.

Maybe option b) is better for consistency? is there someone who has general gut feelings for REF?

Reply:
I can't think of anyone else to ask. 
Given the numbers you found, I think going for b) sounds reasonable

@fbenke-pik could you do that (changing `x_adj["REF", , ] <- 1400` to `x_adj["REF", , ] <- 1100`)? 
I guess it needs a new PR, or can we simply include it in this one?

Reply:
(done in this commit: https://github.com/pik-piam/mrremind/commit/6e894af7a61983e105e6bcc868e9cc6d3ef0b7d7)